### PR TITLE
[AssetMapper] [WCM] Replace ImportMapConfigReader by interface

### PR DESCRIPTION
**Pull request by damienfern**
Related to this PR : https://github.com/symfony/symfony/pull/53485

### CODE REVIEW

File: frontend/asset_mapper.rst [hunk 1]
```diff
@@ -955,6 +955,21 @@ rendered by the ``{{ importmap() }}`` Twig function:
             importmap_script_attributes:
                 crossorigin: 'anonymous'
 
+``framework.asset_mapper.config_reader_id``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Sometimes you may want to create your own config reader to load your importmap or use a custom one from a 3rd party package. You can do this by creating a service that implements ``ImportMapConfigReaderInterface`` and then setting the ``config_reader_id`` option:
```
Comment (alexandre-daubois) [inline frontend/asset_mapper.rst]: Try to break your lines at char 80

File: frontend/asset_mapper.rst [hunk 2]
```diff
@@ -955,6 +955,21 @@ rendered by the ``{{ importmap() }}`` Twig function:
...
+.. code-block:: yaml
+
+    framework:
+        asset_mapper:
+            config_reader_id: App\MyCustomImportMapConfigReader
+
+.. tip::
+
+    Don't forget to define your service in your ``services.yaml`` file if further configuration is needed.
+
```
Comment (alexandre-daubois) [inline frontend/asset_mapper.rst:979]: You may add this at the end of your option description:

```rst
.. versionadded:: 7.1

    The ``framework.asset_mapper.config_reader_id`` was introduced in Symfony 7.1.
```

**Review by alexandre-daubois — CHANGES_REQUESTED** [review]

File: frontend/asset_mapper.rst (hunk 1 above)
Comment (alexandre-daubois) [inline frontend/asset_mapper.rst]:
```suggestion
Sometimes you may want to create your own config reader to load your importmap or use a custom one from a third party package. You can do this by creating a service that implements ``ImportMapConfigReaderInterface`` and then setting the ``config_reader_id`` option:
```

**Comment by javiereguiluz** [comment]
Closing because the related code PR #53485 was closed as "won't fix".

**Pull request closed without merging**